alphadia/calibration/property.py

```diff
@@ -172,7 +172,7 @@ def fit(self, dataframe: pd.DataFrame, plot: bool = False, **kwargs):
             self.function.fit(input_values, target_value)
             self.is_fitted = True
         except Exception as e:
-            logging.error(f"Could not fit estimator {self.name}: {e}")
+            logging.exception(f"Could not fit estimator {self.name}: {e}")
```

Comment:
This will take some time until it reaches muscle memory :D

Reply:
doesn't need to: it's an autofix :-D